src/Utilities/functions.jl

```diff
@@ -121,6 +120,15 @@ function Base.getindex(it::ScalarFunctionIterator{MOI.VectorAffineFunction{T}},
     MOI.VectorAffineFunction(terms, constant)
 end
 
+termindices(t::MOI.ScalarAffineTerm) = (t.variable_index.value,)
+termindices(t::MOI.VectorAffineTerm) = (t.output_index, termindices(t.scalar_term)...)
+
+unsafe_add(t1::MOI.ScalarAffineTerm, t2::MOI.ScalarAffineTerm) = MOI.ScalarAffineTerm(t1.coefficient + t2.coefficient, t1.variable_index)
```

Comment:
Document what it does and explains that it is unsafe because it assume that the variable index and output_index is the same

Reply:
Sure